### DISCUSSION TITLE
What are the white dots in some of the graph bars?

### DISCUSSION THREAD

**Discussion by DigiH** [theengs/app#50] [Q&A]
As the title says, I'm curious as to what the white dots in the orange CGDN1 CO2 bars indicate.

Thanks

![IMG_1782](https://user-images.githubusercontent.com/17110652/202206334-d42cf777-194a-42a3-b874-da4c9a743e38.PNG) ![IMG_1783](https://user-images.githubusercontent.com/17110652/202206364-04c0bd75-0396-4b57-a92a-402010433db8.PNG)

**Comment by DigiH** [comment]
And on the IBS-TH1 graphs for that matter? :)

![IMG_1784](https://user-images.githubusercontent.com/17110652/202251245-e39c4612-3f9b-4e67-8ec9-fa6b33aaa036.PNG)

**Reply by emericg** [reply]
Dots are the average values.

**Reply by DigiH** [reply]
This is what I thought at first as well, but looking at the CGDN1 CO2 bars, only one of the orange bars of each device has a dot, and how could the average be so low, even lower than any of the other days green bars.

Also looking and comparing with my controller's detailed graphs for these days the CO2 dots do not make sense at all as an average.

For the green IBS-TH1 temperature bars with the dots in the middle of them it does make more sense and seems fine.

**Reply by DigiH** [reply]
Further watching this I found that the **average** value dot indication varies greatly and very much depends on the amount of samples the app gets. The same for the actual bars themselves.

In case of the mobile app, especially on iOS where no background updates are available at all, new samples only get  added when the app is being opened/brought to the foreground. 

Similarly on macOS, where the app was open longer and continued to receive background updates regularly during that time, the **average** value dots for the same day (15th) are totally different, making it a bit confusing to know which **average** values to trust. 

And why haven't the macOS orange bars for the 16th a white **average** dot as well?

Just trying to get behind the functionality of these indicators.

<img width="722" alt="Screenshot 2022-11-21 at 15 16 37" src="https://user-images.githubusercontent.com/17110652/203077666-0256104c-ac80-4531-b1ed-14a10a09284c.png">
<img width="719" alt="Screenshot 2022-11-21 at 15 16 24" src="https://user-images.githubusercontent.com/17110652/203077681-a7c38bf5-c5e4-4035-9cb2-618430779abd.png">
